.github/actions/vmtest/action.yml

```diff
@@ -0,0 +1,32 @@
+name: 'vmtest'
+description: 'Build + run vmtest'
+runs:
+  using: "composite"
+  steps:
+    # 1. Setup environment
+    - name: Setup build environment
+      uses: libbpf/ci/setup-build-env@master
+    # 2. Build
+    - name: Build kernel image
+      shell: bash
+      run: |
```

Comment:
nit: this probably always going to be single shell script invocation, so "|" is unnecessary, right? We can keep it terse and single-line?